.github/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+# This should match the owning team set up in https://github.com/orgs/opensearch-project/teams
+*   @opensearch-project/opensearch-core @reta
```

Comment:
@CEHENKLE apologies, could you please add `@opensearch-project/opensearch-core` to maintainers (I don't have enough perms), thank you